book/src/dev/release-process.md

```diff
@@ -0,0 +1,170 @@
+# Zebra versioning and releases
+
+This document contains the practices that we follow to provide you with a leading-edge application, balanced and with stability.
+We strive to ensure that future changes are always introduced in a predictable way.
+We want everyone who depends on Zebra to know when and how new features are added, and to be well-prepared when obsolete ones are removed.
+
+<a id="versioning"></a>
+
+## Zebra versioning
+
+Zebra version numbers indicate the level of changes that are introduced by the release.
+This use of [semantic versioning](https://semver.org/ "Semantic Versioning Specification") helps you understand the potential impact of updating to a new version.
+
+Zebra version numbers have three parts: `major.minor.patch`.
+For example, version `3.1.11` indicates major version 3, minor version 1, and patch level 11.
+
+The version number is incremented based on the level of change included in the release.
+
+<div class="alert pre-release">
+
+**NOTE**: <br />
+As Zebra is in a `pre-release` state (is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version).
+The pre-release version is denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version.
+
+</div>
+
+| Level of change | Details |
+|:---             |:---     |
+| Major release   | Contains significant new features, and commonly correspond to network upgrades; some technical assistance is expected during the update. When updating to a new major release, you might need to <fill-with-things-the-user-might-need-to-do>. |
+| Minor release   | Contains new smaller features. Minor releases are meant to be fully backward-compatible, but may include breaking changes to public APIs. No technical assistance is expected during update. |
+| Patch release   | Low risk, bug fix release. No technical assistance is expected during update. |
+
```

Comment:
We need some experience during the release candidate series before we can decide how long to support releases.

But we don't want people thinking we support every Zebra version forever, so we still need to provide some minimum support periods. (It is much easier to extend supported releases than remove them.)

```suggestion

<a id="supported-releases"></a>

### Supported Releases

Older Zebra versions are always supported until:
* the next Zebra major, minor or patch release.

Initially, we can only guarantee support for the latest Zebra release. We might be able to provide support for earlier releases, or we might ask you to upgrade to the latest release.

Our support periods will be extended as we gain experience with supporting Zebra releases.

Older Zebra versions that only support previous network upgrades will never be supported, because they are operating on an unsupported Zcash chain fork.

```